docs/api.md

```diff
@@ -172,6 +172,12 @@ The sub-fields are
 * *rotational* -- A boolean indicating whether the device should be
   a rotating disk (`true`) or not (`false`).
 
+#### automatedCleaningMode
+
+An interface to enable/disable automated cleaning during provisioning
+and deprovisioning. When set to `Disabled`, automated cleaning will be
+skipped, where `Metadata`(default value) enables it.
```

Comment:
How do we expect 'Metadata' to differ from other possible cleaning modes?

Reply:
This was decided to be called Metadata to reflect Ironic Metadata cleaning after the discussion [here](https://github.com/metal3-io/baremetal-operator/pull/784#discussion_r589422010). So, in the future there might be on option to select only certain devices to be cleaned.